modules/ROOT/pages/batch-error-handling-faq.adoc

```diff
@@ -40,19 +49,7 @@ The error summary for a batch job with two batch steps that raised a `batch.exce
 ************************************************************************************************************************
 ----
 +
-Here we can see that the first step failed ten times and the second failed nine.
-
-Mule logs batch errors using this behavior as its default configuration, which only logs INFO level messages to get a balanced trade-off between logging efficiency for large chunks of data. +
-However, your motivation to use batch processing might not be its ability to handle large datasets, but its capacity to recover from a crash and keep processing a batch job from where it was left off when performing "near real-time" data integration. Or you may need more verbose log levels for debugging. +
-For those cases, you can set the batch category to DEBUG level in your application's log4j configuration like this:
-
-[source,text,linenums]
-----
-log4j.logger.com.mulesoft.module.batch=DEBUG
-----
-
-[CAUTION]
-Use the DEBUG level with care. Definitively not advised for production environments unless you’re sure that your jobs will not handle significantly large data sets.
+Here, the first step failed ten times, and the second failed nine.
```

Comment:
I don't think the second comma is needed, but probably fine.

Reply:
next time, thx